lib/config.js

```diff
@@ -343,7 +343,7 @@ class Config {
     }
     this.browserDisconnectTimeout = 2000
     this.browserDisconnectTolerance = 0
-    this.browserNoActivityTimeout = 20000
```

Comment:
the previous default was 20s?

Reply:
yea, in #3087 two days ago I set it to 20s. Then I saw the travis page recommending 30s. bah.